Library/Formula/exomizer.rb

```diff
@@ -0,0 +1,19 @@
+class Exomizer < Formula
+  desc     "6502 compressor with CBM PET 4032 support."
```

Comment:
Please don't equalise the indent; It gets flagged by various editors and it's not worth the hassle. Just use two spaces for everything.